terminus/geometry/arc.py

```diff
@@ -195,7 +242,7 @@ def heading_at_offset(self, offset):
         if offset > self.length():
             raise ValueError("Offset ({0}) is greater than segment length ({1})".format(offset, self.length()))
         angular_offset = math.copysign(offset, self._angular_length) * 180 / (math.pi * self._radius)
-        return self._theta + angular_offset
+        return (self._theta + angular_offset) % 360
```

Comment:
yes, but in the future the heading will be a vector, not an angle. I just want to minimize the places we have to search for `% 360` and update to whatever we use when we use a direction vector. That is why, instead of introducing `% 360` in a bunch of places I prefer it to be in a single method that is called when needed.